spec/models/supplejack_api/user_set_spec.rb

```diff
@@ -43,6 +43,12 @@ module SupplejackApi
         expect(user_set.privacy).to eq "public"
       end
 
+      it "sets the subjects field to [] if it is nil" do
+        user_set.subjects = nil
+        user_set.save
+        user_set.reload
+        expect(user_set.subjects).to eq []
```

Comment:
I would put a space between the test set up and the expect. Its just a styling thing but it is good practice to give your tests some breathing room